game-core/gradle.properties

```diff
@@ -1,3 +1,3 @@
 checkstyleIntegTestMaxWarnings=0
-checkstyleMainMaxWarnings=1294
-checkstyleTestMaxWarnings=2
+checkstyleMainMaxWarnings=1592
+checkstyleTestMaxWarnings=4
```

Comment:
The checkstyle update caused new javadoc related warnings.
We can disabled the warnings in a follow-up PR if this is desireable